staging/kos/pkg/controllers/connectionagent/connection_agent.go

```diff
@@ -687,8 +736,35 @@ func (ca *ConnectionAgent) processQueueItem(attNSN k8stypes.NamespacedName, qlen
 	ca.queue.Forget(attNSN)
 }
 
+func (ca *ConnectionAgent) getLastClientWriteInfo(att *netv1a1.NetworkAttachment,
+	vnRelevanceTrigger string,
+	vnRelevanceTime time.Time) (lastClientWr string, lastClientWrTime time.Time) {
+
+	if att == nil {
+		return
+	}
+
+	lastClientWr, lastClientWrTime = getAttLastClientWriteInfo(att, naLastClientWrite, subnetLastClientWrite)
+	if att.Spec.Node != ca.node && vnRelevanceTime.After(lastClientWrTime) {
+		lastClientWr = vnRelevanceTrigger
+		lastClientWrTime = vnRelevanceTime
+	}
+
+	return
+}
+
+func getAttLastClientWriteInfo(att *netv1a1.NetworkAttachment, naLastClientWrVal, naSubnetLastClientWrVal string) (lastClientWr string, lastClientWrTime time.Time) {
```

Comment:
The name of this func, "getAttLastClientWriteInfo", when compared with the preceding "getLastClientWriteInfo", differs only in the presence of "Att" --- which does not identify the difference between these two funcs.  Perhaps this one should be named something more distintive.  Perhaps something like "lastClientWriteInfoSwitch"?

Reply:
The reasoning behind the name "getAttLastClientWriteInfo" is that it returns the last client write concerning an attachment, whereas the enclosing func returns the last client write among a group of writes that is related not just to the attachment being processed.

Reply:
Both are rooted at the NA, and even `getAttLastClientWriteInfo` looks further.
It is not just how far the func looks, but `getAttLastClientWriteInfo` takes parameters to use in characterizing the answer.